pkg/geo/geogfn/unary_operators.go

```diff
@@ -128,3 +166,65 @@ func length(regions []s2.Region, useSphereOrSpheroid UseSphereOrSpheroid) (float
 	}
 	return totalLength, nil
 }
+
+// longitudeNormalize convert a longitude to the range of -Pi, Pi.
+func longitudeNormalize(lon float64) float64 {
```

Comment:
can we convert this to `s1.Angle`, and use `s1.Angle(blah).Normalized()`?

Reply:
Done.